content/v2.0/write-data/use-telegraf/use-http-plugin.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Use the HTTP input plugin with Citi Bike data
+description:
+menu:
+  v2_0:
+    parent: Use Telegraf
+weight: 202
+---
+
+This example walks through using the Telegraf HTTP input plugin to collect live metrics on Citi Bike stations in New York City. Live station data is available in JSON format from [NYC OpenData](https://data.cityofnewyork.us/NYC-BigApps/Citi-Bike-Live-Station-Feed-JSON-/p94q-8hxh).
+
+For the following example to work, configure [`influxdb` output plugin](telegraf/v1.13/plugins/plugin-list/#influxdb). This plugin is what allows Telegraf to write the metrics to your InfluxDB.
+
+## Configure the HTTP Input plugin in your Telegraf configuration file
+
+To retrieve data from the Citi Bike URL endpoint, enable the `inputs.http` input plugin in your Telegraf configuration file.
+
+Specify the following options:
+
+### `urls`
+One or more URLs to read metrics from. For this example,  use `https://feeds.citibikenyc.com/stations/stations.json`.
+
+### `data_format`
+The format of the data in the HTTP endpoints that Telegraf will ingest. For this example, use JSON.
+
+
+## Add parser information to your Telegraf configuration
+
+Specify the following JSON-specific options.
+
+### JSON
+
+#### `json_query`
+To parse only the relevant portion of JSON data, set the `json_query` option with a [GJSON](https://github.com/tidwall/gjson) path. The result of the query should contain a JSON object or an array of objects.
+In this case, we don't want to parse the JSON query's `executionTime` at the beginning of the data, so we'll limit this to include only the data in the `stationBeanList` array.
+
+#### `tag_keys`
+List of one or more JSON keys that should be added as tags. For this example, we'll use the tag keys `id`, `stationName`, `city`, and `postalCode`.
+
+#### `json_string_fields`
+List the keys of fields that are in string format so that they can be parsed as strings. Here, the string fields are `statusValue`, `stAddress1`, `stAddress2`, `location`, and `landMark`.
+
+#### `json_time_key`
+Key from the JSON file that creates the timestamp metric. In this case, we want to use the time that station data was last reported, or the `lastCommunicationTime`. If you don't specify a key, the time that Telegraf reads the data becomes the timestamp.
+
+#### `json_time_format`
+The format used to interpret the designated `json_time_key`. This example uses [Go reference time format](https://golang.org/pkg/time/#Time.Format). For example, `Mon Jan 2 15:04:05 MST 2006`.
+
+#### `json_timezone`
+The timezone We'll set this to the Unix TZ value where our bike data takes place, `America/New_York`.
```

Comment:
I don't know if we need to outline all of this here. I think most people will probably just want to copy and paste the `telegraf.conf` toml.

Reply:
I'm going to leave it in because I think it's helpful to walk through each options in case they want to adapt this tutorial to another data source.